src/index.ts

```diff
@@ -381,14 +381,12 @@ export const useForm = <Values extends Record<string, any>, ErrorMessage = strin
     };
 
     const getOnBlur = (name: Name) => (): void => {
-      if (isTalkative(name)) {
-        return; // Avoid validating a field validated on each change
-      }
-
-      setTalkative(name, ["onFirstBlur", "onFirstSuccessOrFirstBlur"]);
+      const { validity } = states.current[name];
 
-      if (isMounted(name)) {
-        internalValidateField(name);
+      // Avoid validating an untouched / already valid field
+      if (validity.type !== "unknown" && !isTalkative(name)) {
+        setTalkative(name, ["onFirstBlur", "onFirstSuccessOrFirstBlur"]);
+        isMounted(name) && internalValidateField(name);
```

Comment:
Just a little detail: I prefer use the previous pattern for conditional function call (I think because I'm not used to use the pattern `condition && callFunction()`).